src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj

```diff
@@ -1,14 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <RootNamespace>Microsoft.AspNetCore</RootNamespace>
     <VerifyAncmBinary Condition="'$(TargetOsName)' == 'win' AND '$(TargetArchitecture)' != 'arm'">true</VerifyAncmBinary>
-    <TestDependsOnAspNetPackages>true</TestDependsOnAspNetPackages>
```

Comment:
One thing to consider is to just stop running these special app tests on helix if these are now the only ones that require all this custom packaging, and just run these on the build agents once (say the windows test job).  Might save us a bunch of grief in the long term maintaining this stuff if we no longer need it for any of the other normal tests

Reply:
> One thing to consider is to just stop running these special app tests on helix if these are now the only ones that require all this custom packaging, and just run these on the build agents once (say the windows test job). Might save us a bunch of grief in the long term maintaining this stuff if we no longer need it for any of the other normal tests

There's now very little infrastructure for this and any move away from Helix is a regression. In addition, the project template tests need more stuff than these tests.